src/components/Common/InternalError/InternalError.module.scss

```
@@ -0,0 +1,27 @@
+h1 {
+  font-size: toRem(21) !important;
+}
+
+.internalErrorCard {
+  background-color: #fff;
+  border-right: 1px solid #eaeaea;
+  border-bottom: 1px solid #eaeaea;
+  border-left: 1px solid #eaeaea;
+  box-shadow:
+    0px 1px 18px rgba(0, 0, 0, 0.08),
+    inset 0 4px 0 #d5351f;
+  padding: toRem(20);
+  display: flex;
+  flex-direction: column;
+  gap: toRem(8);
+  max-width: toRem(600);
+  margin: 0 auto;
+}
+
+.errorMessage {
+  font-size: toRem(14);
+  font-weight: 500;
```

We can use a variable here 

```suggestion
  font-weight: var(--g-typography-fontWeight-medium);
```